ppr-ui/src/composables/mhrInformation/useTransferOwners.ts

```diff
@@ -47,7 +47,7 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
   const {
     getGroupById,
     getCurrentGroupById
-  } = useHomeOwners(!!getMhrTransferType.value)
+  } = useHomeOwners(true)
```

Comment:
@cameron-eyds, you mentioned this could cause errors. We should discuss this.

Reply:
Thanks for the breakdown. I agree, we should remove the useHomeOwners from the transfers composable. I will ask Patrick to create a separate ticket for it. Meanwhile, could you please block this PR from merging by 'Requesting Change', thanks!

Reply:
I did this locally in about 10 minutes, do you want me to commit this? 
Wouldn't it make sense to include this here, so you could test again your current changes?

Reply:
If you want, reference this code change: https://github.com/bcgov/ppr/compare/main...cameron-eyds:ppr:Composable-refactor

Or if there is a new ticket created, let me know and i can put this up for PR. or whatever works 👍

Reply:
Thanks for the reference, updated!

Reply:
Right on!